.github/workflows/wheel-builder.yml

```diff
@@ -378,10 +410,13 @@ jobs:
               PY_LIMITED_API="--config-settings=build-args=--features=pyo3/abi3-${{ matrix.PYTHON.ABI_VERSION }}"
           fi
 
-          uv build --wheel --require-hashes --build-constraint=$BUILD_REQUIREMENTS_PATH cryptography*.tar.gz $PY_LIMITED_API -o wheelhouse/
+          echo $PY_LIMITED_API
+          ls cryptography*.tar.gz
+
+          uv build --python='${{ steps.setup-python.outputs.python-path }}' -v --wheel --require-hashes --build-constraint=$BUILD_REQUIREMENTS_PATH cryptography*.tar.gz $PY_LIMITED_API -o wheelhouse/
```

Comment:
We can undo this now.

Reply:
This is still needed because of the uv solver issue I explained about in my earlier comment.